Tests/TuistDependenciesTests/SwiftPackageManager/Utils/PackageInfoMapperTests.swift

```diff
@@ -1857,8 +1879,34 @@ extension ProjectDescription.Target {
         headers: ProjectDescription.Headers? = nil,
         dependencies: [ProjectDescription.TargetDependency] = [],
         customSettings: ProjectDescription.SettingsDictionary = [:],
-        moduleMap: String? = nil
+        moduleMap: String? = nil,
+        excludedResources: [String] = [],
+        sourcesPath: String? = nil
     ) -> Self {
+        let defaultResourceBasePath: String
+        if let sourcesPath = sourcesPath {
+            let relativePath = RelativePath("Package/Path/\(sourcesPath)/**")
+            defaultResourceBasePath = basePath.appending(relativePath).pathString
+        } else {
+            defaultResourceBasePath = basePath.appending(RelativePath("Package/Path/Sources/\(name)/**")).pathString
+        }
+
+
+        let defaultResources = ["xib", "storyboard", "xcdatamodeld", "xcmappingmodel", "xcassets", "lproj"]
+            .map { file -> ProjectDescription.ResourceFileElement in
+                return ResourceFileElement.glob(
+                    pattern: Path("\(defaultResourceBasePath)/*.\(file)"),
+                    excluding: excludedResources.map(Path.init(stringLiteral:))
+                )
+            }
+
+        let targetResources: ResourceFileElements
+        if let resources = resources {
+            targetResources = ResourceFileElements(resources: resources.resources + defaultResources)
```

Comment:
I would change the resources parameter to be a [ResourceFileElements] defaulting to [] so we don't need to handle this difference here